docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
     build:
       context: .
       dockerfile: ./Docker/Dockerfile.worker
+    image: judge_worker_v2
```

Comment:
This should be just `judge_worker` without v2

Reply:
Done.